app/Helpers/database/achievement.php

```diff
@@ -359,6 +362,10 @@ function UploadNewAchievement(
             static_setlastupdatedgame($gameID);
             static_setlastupdatedachievement($idInOut);
 
+            if ($changingLogic) {
+                $achievement->upsertAuthorshipCredit($author, AchievementAuthorTask::LOGIC);
+            }
```

Comment:
If a developer modifies the logic of an achievement, we also give the developer a logic task credit record.

It is easy enough to also give writing credit here if the title or description are being edited, but I decided to be a bit conservative and not do this, as it's possible someone is just fixing a typo and I wasn't sure if that constituted "writing credit". This isn't a strong opinion and I could just as easily go the other way, though.

Reply:
Same comment as below: Should the logic credit track the first edit by the user, or the last?

Reply:
Personally, I think it should track the first. Updated in latest.